devbox/common/src/Rpc.scala

```diff
@@ -12,7 +12,8 @@ object Action{
 }
 
 object Rpc{
-  case class FullScan(roots: Seq[os.RelPath]) extends Rpc
+  /** Perform a full scan of each root path, including any gitignored files appearing in the cooresponding forceIncludes */
+  case class FullScan(roots: Seq[os.RelPath], forceIncludes: Seq[Seq[os.SubPath]], syncGit: Boolean) extends Rpc
```

Comment:
Let's flip `syncGit` into `proxyGit`, just for consistency with the command line flag